upup/models/bindata.go

```diff
@@ -12873,6 +12873,9 @@ spec:
             # Enable Prometheus process metrics collection
             - name: FELIX_PROMETHEUSPROCESSMETRICSENABLED
               value: "{{- or .Networking.Calico.PrometheusProcessMetricsEnabled "true" }}"
+            # Enable / Disable source/destination checks in AWS
+            - name: FELIX_AWSSRCDSTCHECK
+              value: "{{- if and (eq .CloudProvider "aws") (.Networking.Calico.CrossSubnet) -}}Disable{{- else -}} {{- or .Networking.Calico.AwsSrcDstCheck "DoNothing" -}} {{- end -}}"
```

Comment:
Here we tell Calico to take control of the ENI source/destination checks if "CrossSubnet" is in effect, but we don't check whether "AwsSrcDstCheck" was "DoNothing." If the user sets "CrossSubnet" but fails to set "AwsSrcDstCheck," then Calico attempts to reconfigure the ENI, but its governing IAM role lacks permission to do so.